core/cmd/app.go

```diff
@@ -31,11 +35,45 @@ func NewApp(client *Client) *cli.App {
 			Name:  "json, j",
 			Usage: "json output as opposed to table",
 		},
+		cli.StringFlag{
+			Name:  "admin-credentials-file",
+			Usage: "optional, applies only in client mode when making remote API calls. If provided, `FILE` containing admin credentials will be used for logging in, allowing to avoid an additional login step. If `FILE` is missing, it will be ignored",
+			Value: filepath.Join(client.Config.RootDir(), "apicredentials"),
+		},
+		cli.StringFlag{
+			Name:  "remote-node-url",
+			Usage: "optional, applies only in client mode when making remote API calls. If provided, `URL` will be used as the remote Chainlink API endpoint",
+			Value: "http://localhost:6688",
+		},
+		cli.BoolFlag{
+			Name:  "insecure-skip-verify",
+			Usage: "optional, applies only in client mode when making remote API calls. If turned on, SSL certificate verification will be disabled. This is mostly useful for people who want to use Chainlink with a self-signed TLS certificate",
+		},
 	}
 	app.Before = func(c *cli.Context) error {
 		if c.Bool("json") {
 			client.Renderer = RendererJSON{Writer: os.Stdout}
 		}
+		urlStr := c.String("remote-node-url")
+		remoteNodeURL, err := url.Parse(urlStr)
+		if err != nil {
+			return errors.Wrapf(err, "%s is not a valid URL", urlStr)
+		}
+		clientOpts := ClientOpts{RemoteNodeURL: *remoteNodeURL, InsecureSkipVerify: c.Bool("insecure-skip-verify")}
+		cookieAuth := NewSessionCookieAuthenticator(clientOpts, DiskCookieStore{Config: client.Config}, client.Logger)
+		sr := sessions.SessionRequest{}
+		sessionRequestBuilder := NewFileSessionRequestBuilder(client.Logger)
+		{
+			credentialsFile := c.String("admin-credentials-file")
+			var err error
+			sr, err = sessionRequestBuilder.Build(credentialsFile)
+			if err != nil && !errors.Is(errors.Cause(err), ErrNoCredentialFile) && !os.IsNotExist(err) {
+				client.Logger.Fatalw("Error loading API credentials", "error", err, "credentialsFile", credentialsFile)
```

Comment:
Is it important to Fatal exit directly, rather than return the error? Looks like it would be logged error level normally: `client.Logger.ErrorIf(app.Run(args), "Error running app")`